doc/man3/ECDSA_SIG_new.pod

```diff
@@ -69,6 +70,10 @@ after this function has been called.
 See L<i2d_ECDSA_SIG(3)> and L<d2i_ECDSA_SIG(3)> for information about encoding
 and decoding ECDSA signatures to/from DER.
 
+All of the functions described below are deprecated. Applications should
+use the higher level B<EVP> interface such as L<EVP_DigestSignInit(3)>
+or L<EVP_DigestVerifyInit(3)> instead.
+
 ECDSA_size() returns the maximum length of a DER encoded ECDSA signature
 created with the private EC key B<eckey>.
```

Comment:
You might want to document recommended methods to get the actual signature size

Reply:
Added a sentence suggesting `EVP_PKEY_sign(3)` with a NULL signature parameter.